plugin/skills/commit-context/SKILL.md

```
@@ -1,19 +1,64 @@
-Present the result as:
-- The commit SHA, short SHA, branch, author, message
-- The linked session(s): id, project, started/ended timestamps, observation count, summary if any
-- A short list of the most important observations from that session (importance >= 7) when available via `memory_recall`
+```json
+memory_commit_lookup { "sha": "9a1b2c3d4e5f60718293a4b5c6d7e8f901234567" }
+```
 
-Do not fabricate intent. If the commit has no linked session, say so plainly and surface only what `git show` reveals. If `memory_commit_lookup` returns an empty `commit: null` body, that means the commit predates session linking — do not invent a session.
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)